Form-Controls/styles.css

```diff
@@ -0,0 +1,46 @@
+html, body{
+  background-color: #ffeeea;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+#button{
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: center;
+  justify-content: space-evenly;
+  margin-right: 1.5rem;
+  
+}
+.btn1{
```

Comment:
Choose names that accurately describe the purpose or content of the element. This makes it easier for others (and your future self) to understand the code. I don't know which `btn` you mean by `btn1` for example.